src/Phinx/Console/Command/Create.php

```diff
@@ -50,6 +58,13 @@ protected function configure()
                 PHP_EOL,
                 PHP_EOL
             ));
+
+        // An alternative template.
+        $this->addOption('template', 't', InputOption::VALUE_REQUIRED, 'Use an alternative template');
+
+        // A classname to be used to gain access to the template content as well as the ability to
+        // have a callback once the migration file has been created.
+        $this->addOption('class', 'l', InputOption::VALUE_REQUIRED, 'Use a class implementing "' . self::CREATION_INTERFACE . '" to generate the template');
```

Comment:
The addition of these two options should be done in a separate pull request. The ability to define a base class that exists outside of `Phinx` namespace is a separate issue from having a `CreationInterface`. Please make this PR deal with the namespace issue only. Thanks!